### ISSUE TITLE
Optimizing Autoencoder

### ISSUE THREAD

**Issue by rm1003** [ferbcastro/ALHO#26]
## Some of the most common hyperparameters that can be tuned when optimizing your Autoencoder are:
 
- The number of layers for the Encoder and Decoder neural networks
- The number of nodes for each of these layers
- The loss function to use for the optimization process (e.g., binary cross-entropy or mean squared error)
- The size of the latent space (the smaller, the higher the compression, acting, therefore as a regularization mechanism)

https://www.datacamp.com/tutorial/introduction-to-autoencoders
